code/modules/mob/living/carbon/human/species_types/dullahan.dm

```diff
@@ -24,17 +24,20 @@
 
 /datum/species/dullahan/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
 	. = ..()
-	H.flags_1 &= ~HEAR_1
+	DISABLE_BITFIELD(H.flags_1, HEAR_1)
```

Comment:
This macro is bad, don't use it

Reply:
Not gonna lie, I saw this coming. Yet I hate tildes.